sops/__init__.py

```diff
@@ -95,6 +95,8 @@
 
 INPUT_VERSION = VERSION
 
+DEFAULT_UNENCRYPTED_SUFFIX = '_unencrypted'
```

Comment:
Up to you how you want to handle it. You could keep the default global, and create another global to store the current value.